rllib/algorithms/bc/bc.py

```diff
@@ -52,11 +66,60 @@ def __init__(self, algo_class=None):
         # Advantages (calculated during postprocessing)
         # not important for behavioral cloning.
         self.postprocess_inputs = False
+        # Important to be False for dummy runs.
+        self.use_gae = False
```

Comment:
Hey @simonsays1980 , one last question, before we can merge: It seems like use_gae is not used at all anywhere, neither by MARWIL, nor by BC. Should we not completely remove this setting then? Or am I missing something?

Reply:
See this comment here in the MARWIL code:
```
            # We just want the discounted cumulative rewards, so we won't need
            # GAE nor critic (use_critic=True: Subtract vf-estimates from returns).
            use_gae=False,
```
https://github.com/ray-project/ray/blob/master/rllib/algorithms/marwil/marwil_tf_policy.py#L68

Reply:
Great catch. I stumbled over it when studyxing the config and thought that the user should have the option to switch over to GAE if needed. B ut you are right, it's never used again. I will erase it.